scripts/qa_crowbarsetup.sh

```diff
@@ -3346,11 +3346,17 @@ function oncontroller_run_tempest
     fi
     if [ -n "$ostestroptions" ]; then
         zypper -n in python-os-testr
+        if [[ $want_magnum = 1 ]]; then
+            pushd /var/lib/openstack-magnum-test
```

Comment:
I understand, but you're overloading ostestroptions with a completely different meaning than what is documented in the openstack mkcloud documentation. 

does magnum provide a tempest plugin? in that case these switches are not needed. if it doesn't, maybe this should be moved into cct directly instead.